doc/en/user/source/geowebcache/webadmin/layers.rst

```diff
@@ -63,8 +63,22 @@ To add in a layer from GeoServer (if it wasn't set up to be added automatically)
 .. figure:: img/newcachedlayer.png
 
    Adding a new cached layer
+   
+Clearing GeoWebCache
+---------------------------
+
+To wipe the GeoWebCache clean, instead of clicking Empty for each layers one at a time, simple click `Empty All` and confirm.
```

Comment:
"Wipe clean" is probably too strong of a wording. "Clean up the entire cache, for all layers, gridsets and filter parameters" might be more descriptive.

```suggestion
The ``Empty all`` link allow to clear the entire cache, for all layers, grid sets and filter parameters combination.
```
Also be consistent, the title should be just "Empty all"

Reply:
Ah, do similar changes wherever "wipe" was used.

Reply:
will change